Add-ons/UmbracoForms/Installation/Install/index-v9.md

```diff
@@ -0,0 +1,38 @@
+---
+versionFrom: 9.0.0
+
+meta.Title: "Installing Umbraco Forms"
+meta.Description: "Installing Umbraco Forms"
+---
+
+# Extending Umbraco with the Forms Section
+
+Since Umbraco v7.2, Umbraco contains the **Forms** section, by default. You will see a similar interface, when you click on the **Forms** section in the **Umbraco Backoffice**.
+
+![Form section in backoffice](images/Forms_Section_Backoffice.png)
+
+## Installing Umbraco Forms
+
+To install the Umbraco Forms package (**Umbraco.Forms.9.0.0-beta001**), follow these steps:
```

Comment:
Would suggest not mentioning the version here, otherwise we'd need to update the docs every time a new version is released.  Think we can omit this, and people will get the latest version, and if they want a specific one they can add the version parameter.

```suggestion
To install the Umbraco Forms package (**Umbraco.Forms**), follow these steps:
```

Reply:
Got it, removing the version